basiclibrary/src/main/java/Library.java

```diff
@@ -15,16 +19,54 @@
         return rollArray;
     }
 
-    public static boolean containsDuplicates(int[] rollArrayToCheckForDuplicates) {
-        for (int i = 0; i < rollArrayToCheckForDuplicates.length; i++) {
-            for (int j = i + 1; j < rollArrayToCheckForDuplicates.length; j++) {
-                if (rollArrayToCheckForDuplicates[i] == (rollArrayToCheckForDuplicates[j])) {
+    public static boolean containsDuplicates(int[] arrayToCheckDuplicates) {
+        for (int i = 0; i < arrayToCheckDuplicates.length - 1; i++) {
+            for (int j = i + 1; j < arrayToCheckDuplicates.length; j++) {
+                if (arrayToCheckDuplicates[i] == (arrayToCheckDuplicates[j])) {
                     return true;
                 }
             }
         }
         return false;
     }
 
+    public static void minMaxInTwoDimArray() {
+        int[][] weeklyMonthTemperatures = {
+                {66, 64, 58, 65, 71, 57, 60},
+                {57, 65, 65, 70, 72, 65, 51},
+                {55, 54, 60, 53, 59, 57, 61},
+                {65, 56, 55, 52, 55, 62, 57}
+        };
+        minMaxInTwoDimArray(weeklyMonthTemperatures);
+
+        Set<Integer> targetTempSet = new HashSet<Integer>();
+
+        for (int i = 0; i < weeklyMonthTemperatures.length; i++) {
+            for (int j = 0; j < weeklyMonthTemperatures[i].length; j++) {
+                if ( !targetTempSet.contains(weeklyMonthTemperatures[i][j]) ) {
+                    targetTempSet.add(weeklyMonthTemperatures[i][j]);
+                }
+            }
+            System.out.println(targetTempSet);
+
+            return targetTempSet;
```

Comment:
You have a return in a void method that won't work. Remember, void method do not return anything. Please reconsider this method - perhaps you don't want it to be void? or you just want it to print out lines?